ext/gd/gd.stub.php

```diff
@@ -100,156 +101,157 @@ function imagecreatefrombmp(string $filename): GdImage|false {}
 
-function imagecopy(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_w, int $src_h): bool {}
+function imagecopy(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_width, int $src_height): bool {}
 
-function imagecopymerge(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_w, int $src_h, int $pct): bool {}
+function imagecopymerge(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_width, int $src_height, int $pct): bool {}
 
-function imagecopymergegray(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_w, int $src_h, int $pct): bool {}
+function imagecopymergegray(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_width, int $src_height, int $pct): bool {}
 
-function imagecopyresized(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_w, int $dst_h, int $src_w, int $src_h): bool {}
+function imagecopyresized(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_width, int $dst_height, int $src_width, int $src_height): bool {}
 
-function imagesx(GdImage $im): int {}
+function imagesx(GdImage $image): int {}
 
-function imagesy(GdImage $im): int {}
+function imagesy(GdImage $image): int {}
 
-function imagesetclip(GdImage $im, int $x1, int $x2, int $y1, int $y2): bool {}
+function imagesetclip(GdImage $image, int $x1, int $x2, int $y1, int $y2): bool {}
 
-function imagegetclip(GdImage $im): array {}
+function imagegetclip(GdImage $image): array {}
 
 #ifdef HAVE_GD_FREETYPE
-function imageftbbox(float $size, float $angle, string $font_file, string $text, array $extrainfo = []): array|false {}
+function imageftbbox(float $size, float $angle, string $font_filename, string $string, array $extra_info = []): array|false {}
 
-function imagefttext(GdImage $im, float $size, float $angle, int $x, int $y, int $col, string $font_file, string $text, array $extrainfo = []): array|false {}
+function imagefttext(GdImage $image, float $size, float $angle, int $x, int $y, int $color, string $font_filename, string $string, array $extra_info = []): array|false {}
 
 /** @alias imageftbbox */
-function imagettfbbox(float $size, float $angle, string $font_file, string $text, array $extrainfo = []): array|false {}
+function imagettfbbox(float $size, float $angle, string $font_filename, string $string, array $extra_info = []): array|false {}
 
 /** @alias imagefttext */
-function imagettftext(GdImage $im, float $size, float $angle, int $x, int $y, int $col, string $font_file, string $text, array $extrainfo = []): array|false {}
+function imagettftext(GdImage $image, float $size, float $angle, int $x, int $y, int $color, string $font_filename, string $string, array $extra_info = []): array|false {}
```

Comment:
I had a similar situation with "message" in intl. Since, the function name used "message", I followed this name with the parameter. So maybe we could use `$text` rather than `$string` (but this is just a similar hair-splitting what we had in oci8, nothing very important).

Reply:
I agree, as the function is imagettftext, it's better to use $text.